### ISSUE TITLE
[Request]

### ISSUE THREAD

**Issue by hamster65** [malnvenshorn/OctoPrint-FilamentManager#109]
I always forget to select the right spool before starting the print. It would be a nice feature if filament manager could ask which spool to use.

**Comment by malnvenshorn** [comment]
You can enable a confirmation dialog in the settings.

Duplicate of #36, closing.
